source/opt/pass_manager.cpp

```diff
@@ -41,9 +49,64 @@ Pass::Status PassManager::Run(ir::IRContext* context) {
     }
   };
 
+#if defined(SPIRV_ANDROID) || defined(SPIRV_LINUX) || defined(SPIRV_MAC) || \
+    defined(SPIRV_FREEBSD)
+  double (*get_time_difference)(const timeval&, const timeval&) = NULL;
+  if (ftime_report_stream_) {
+    *ftime_report_stream_<< std::setw(30) << "Pass Name"
+      << std::setw(16) << "CPU time"
+      << std::setw(16) << "WALL time"
+      << std::setw(16) << "SYS time"
+      << std::setw(16) << "Pagefaults"
+      << std::setw(16) << "RSS" << std::endl;
+    get_time_difference = [](const timeval& before, const timeval& after) -> double {
```

Comment:
This function doesn't need to be a lambda, you can simply define this in anonymous namespace.

Reply:
Done